torch/distributed/optim/zero_redundancy_optimizer.py

```diff
@@ -298,29 +259,30 @@ def consolidate_state_dict(self, to: int = 0) -> None:
                     )
                 else:
                     # Receive the optimizer state from the source rank
-                    local_state_dict = _broadcast_object(
-                        empty_messenger,
+                    object_list = [empty_messenger]
+                    distributed_c10d.broadcast_object_list(
```

Comment:
Same as in the test file: I think `dist.broadcast_object_list()` works, so no need for the additional `import torch.distributed.distributed_c10d as distributed_c10d`.

Reply:
Got it, seems it's enabled by 

distributed/__init__.py 
  from .distributed_c10d import *  # noqa: F403 :)